ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -217,7 +217,7 @@ presubmits:
           value: e2e-restore-cluster-state
         - name: TEST_COMMAND
           value: |
-            restore-cluster-state
+            restore-cluster-state 
```

Comment:
nit: no need to touch this line that I can see (you're adding trailing whitespace).

Reply:
Rehearse test won't trigger without it

Reply:
So we can drop this change once we're happy with the rehearsals?

Reply:
Yeah, this would be dropped once the code is reviewed

Reply:
Removed this commit